### ISSUE TITLE
A t.web glossary is needed

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10781]
|<img alt="tenshihan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| tenshihan reported|
|-|-|
|Trac ID|trac#1176|
|Type|enhancement|
|Created|2005-09-19 06:27:02Z|
|Branch|https://github.com/twisted/twisted/tree/web-glossary-1176|

Attachments:

* [resource.Resource.txt](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a8f/a8f3809c6de28975c83009f6e2911b81f95e5b5a/34baa9231ec772b35d25317cc49c6fc125e7b30c.txt) (562 bytes) - added by tenshihan on 2005-09-19 07:00:24Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__1176 1176
type__enhancement enhancement
reporter__tenshihan tenshihan
priority__normal normal
milestone__ 
branch__branches_web_glossary_1176 branches/web-glossary-1176
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1127111222000000 1127111222000000
changetime__1232117301000000 1232117301000000
version__ 
owner__ 
cc__radix cc__exarkun cc__tenshihan
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="tenshihan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|tenshihan commented|
|-|-|

```
#!html
<pre>
A glossary of terms would be useful in taking the stress of newbs asking
questions in #twisted.web
Being a semi-newb and willing to do this I would like to submit glossary terms
as I learn and write them. I have resource.Resource written.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
This is great! The easiest thing to do would simply be to attach them to this
bug using the File entry box. Plain text is probably the best format. I will
mark them up and commit them.

Also, if you decide to do this for Nevow or if any of your stuff is Nevow
specific, you should submit to their bug tracker (and don't assign to me there).
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="tenshihan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|tenshihan commented|
|-|-|

```
#!html
<pre>
resource.Resource
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement set owner to edsuom|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [25750]) Branching to 'web-glossary-1176'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph set owner to @exarkun|
|-|-|

This looks like a good start.  It's fine to merge, but

 1. it should note that it's incomplete and that additions are welcome (since it is suspiciously short), and
 1. at least one or two other tickets should be filed for jargon twisted-web terms that could be added.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

Replying to [glyph](#note_7):
> This looks like a good start.  It's fine to merge, but
>
>  1. it should note that it's incomplete and that additions are welcome (since it is suspiciously short), and

Fixed in r26042

>  1. at least one or two other tickets should be filed for jargon twisted-web terms that could be added.

Filed [#3620](https://github.com/twisted/twisted/issues/3620).

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set status to `closed`|
|-|-|

(In [26044]) Merge web-glossary-1176

Author: exarkun
Reviewer: glyph
Fixes: [#10781](https://github.com/twisted/twisted/issues/10781)

Add a glossary to Twisted Web.  It currently contains only one term, but it
should be expanded (and hopefully now that it exists, it will be).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
